crnk-rs/src/main/java/io/crnk/rs/JsonApiResponseFilter.java

```diff
@@ -92,4 +101,34 @@ private boolean isResourceResponse(Object response) {
 		boolean resourceList = ResourceListBase.class.isAssignableFrom(response.getClass());
 		return singleResource || resourceList;
 	}
+
+	/**
+	 * Uses {@link JsonApiRequestProcessor#isJsonApiRequest(HttpRequestContext)} to determine if the caller
+	 * requested a response in JSON-API format.
+	 *
+	 * @param requestContext the current container request context as received by this filter
+	 * @return <code>true</code>, if the caller requested a JSON-API response,<br />
+	 *     		<code>false</code>, otherwise
+	 */
+	private boolean isJsonApiRequest(ContainerRequestContext requestContext) {
+		HttpRequestContextBase httpRequestContextBase = new JaxrsRequestContext(requestContext, feature);
+		HttpRequestContextBaseAdapter httpRequestContext = new HttpRequestContextBaseAdapter(httpRequestContextBase);
+		return JsonApiRequestProcessor.isJsonApiRequest(httpRequestContext);
```

Comment:
I think it should not be requested dependant. Because it may confuse developers as as they will see different results depending on whether they look at the services with a browser or do a proper json api ajax call. 

I guess there is a possiblity to specify a the content type in the jaxrs method? and maybe a default could be configured to do either all json-api and specify the exceptions or the other way around.